engine/install/debian.md

```diff
@@ -27,51 +28,53 @@ Raspbian versions:
 - Raspbian Bullseye 11 (stable)
 - Raspbian Buster 10 (oldstable)
 
-Docker Engine is supported on `x86_64` (or `amd64`), `armhf`, and `arm64` architectures.
+Docker Engine is compatible with `x86_64` (or `amd64`), `armhf`, and `arm64`
+architectures.
 
 ### Uninstall old versions
 
-Older versions of Docker were called `docker`, `docker.io`, or `docker-engine`.
-If these are installed, uninstall them:
+Older versions of Docker went by the names of `docker`, `docker.io`, or
+`docker-engine`. Uninstall any such older versions before attempting to install
+a new version:
 
 ```console
 $ sudo apt-get remove docker docker-engine docker.io containerd runc
 ```
 
 It's OK if `apt-get` reports that none of these packages are installed.
 
-The contents of `/var/lib/docker/`, including images, containers, volumes, and
-networks, are preserved. If you do not need to save your existing data, and want to
-start with a clean installation, refer to the [uninstall Docker Engine](#uninstall-docker-engine)
-section at the bottom of this page.
+Images, containers, volumes, and networks stored in `/var/lib/docker/` aren't
+automatically removed when you uninstall Docker. If you want to start with a
+clean installation, and prefer to clean up any existing data, refer to the
+[uninstall Docker Engine](#uninstall-docker-engine) section.
 
 ## Installation methods
 
 You can install Docker Engine in different ways, depending on your needs:
 
-- Most users
-  [set up Docker's repositories](#install-using-the-repository) and install
-  from them, for ease of installation and upgrade tasks. This is the
-  recommended approach, except for Raspbian.
+- Docker Engine comes bundled with
+  [Docker Desktop for Linux](../../desktop/install/linux-install.md). This is
+  the easiest and quickest way to get started.
 
-- Some users download the DEB package and
-  [install it manually](#install-from-a-package) and manage
-  upgrades completely manually. This is useful in situations such as installing
-  Docker on air-gapped systems with no access to the internet.
+- You can also set up and install Docker Engine from
+  [Docker's `apt` repository](#install-using-the-repository).
 
-- In testing and development environments, some users choose to use automated
-  [convenience scripts](#install-using-the-convenience-script) to install Docker.
-  This is currently the only approach for Raspbian.
+- [Install it manually](#install-from-a-package) and manage upgrades completely
```

Comment:
I know this was written before you edited this page, but wonder if we can take this opportunity to remove 'completely'?